lib/project/pro_motion/activities/pm_navigation_activity.rb

```diff
@@ -22,7 +22,7 @@ def on_options_item_selected(item)
     def open_fragment(frag, options={})
       mp frag
       mgr = fragmentManager.beginTransaction
-      mgr.add(@fragment_container.getId, frag, frag.class.to_s)
+      mgr.add(@fragment_container.getId, frag, "screen-#{fragmentManager.getBackStackEntryCount + 1}")
```

Comment:
By tagging each fragment, we can now get them from the stack.

Why Android doesn't just have a `fragmentManager.getFragments` or `fragmentManager.getLastFragment` method I don't know.